tests/test_main.py

```diff
@@ -394,16 +392,8 @@ def test_use_coverage(capsys, filesystem):
     assert result.exit_code == 0
     assert '13/13  🎉 13  ⏰ 0  🤔 0  🙁 0' in repr(result.output)
 
-    # replace the .coverage file content with a non existent path to check if an exception is thrown
-    with open('.coverage', 'r') as f:
-        content = f.read()
-
-    # the new path is linux-based, but it just needs to be wrong
-    new_content = re.sub(r'\"[\w\W][^{]*foo.py\"', '"/test_path/foo.py"', content)
-
-    with open('.coverage', 'w') as f:
-        f.write(new_content)
-
+    # remove existent path to check if an exception is thrown
+    os.unlink(os.path.join(str(filesystem), 'foo.py'))
```

Comment:
I'd prefer os.remove here, but it seems like the docs say this is a synonym provided for unix people.